Biasing/test/resim_ecal_conv_deep.py

```diff
@@ -0,0 +1,46 @@
+from LDMX.Framework import ldmxcfg
+p=ldmxcfg.Process("v14_pn_deep_resim")
+
+from LDMX.Ecal import EcalGeometry
+from LDMX.Hcal import HcalGeometry
+import LDMX.Ecal.ecal_hardcoded_conditions as ecal_conditions
+import LDMX.Hcal.hcal_hardcoded_conditions as hcal_conditions
+from LDMX.Biasing import ecal
+from LDMX.SimCore import generators
+from LDMX.Biasing import filters
+
+det = 'ldmx-det-v14-8gev'
+#mysim = ecal.deep_photo_nuclear(det, generators.single_8gev_e_upstream_tagger(), bias_threshold = 5010., processes=['conv','phot)'], ecal_min_Z = 300.)
+mysim = ecal.deep_photo_nuclear(det, generators.single_8gev_e_upstream_tagger(), bias_threshold = 3000., processes=['conv','phot)'], ecal_min_Z = 400.)
```

Comment:
This definition of a simulator does not match the one in the original simulation which can lead to funky results when re-simulating. Maybe just add a comment explaining that the user should align these two?